File: lc1967.java
public class lc1967 {
//    1967. Number of Strings That Appear as Substrings in Word
    public static void main(String[] args) {
        String patterns[]={"a","b","c"};
        String word="aaaaabbbbb";
        int c=0;
        for (int i =0;i< patterns.length;i++){
            if (patterns[i].length()<=word.length()){
                if (word.contains(patterns[i])){
                    c++;
                }
            }
        }
        System.out.println(c);
    }
}

File: lc1967.py
from typing import List

class Solution:
    def numOfStrings(self, patterns: List[str], word: str) -> int:
        c=0
        for i in patterns:
            if word.__contains__(i):
                c+=1
        return c
print(Solution.numOfStrings(0,["a","abc","bc","d"], "abc"))